internal/finder/versioner_test.go

```diff
@@ -324,3 +329,135 @@ func TestFindCompatibleKubectl(t *testing.T) {
 		})
 	}
 }
+
```

Comment:
I don't like the test from this point on. Has the code been generated by some AI assistant?

I think we can reuse the mocks of finder we already have [here](https://github.com/flavio/kuberlr/blob/main/internal/finder/mock_iFinder.go)